src/levels.rs

```diff
@@ -196,11 +196,13 @@ pub const GLOBALMV: InterPredMode = 2;
 pub const NEARMV: InterPredMode = 1;
 pub const NEARESTMV: InterPredMode = 0;
 
-pub type DRL_PROXIMITY = u8;
-pub const NEARISH_DRL: DRL_PROXIMITY = 3;
-pub const NEAR_DRL: DRL_PROXIMITY = 2;
-pub const NEARER_DRL: DRL_PROXIMITY = 1;
-pub const NEAREST_DRL: DRL_PROXIMITY = 0;
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, FromRepr)]
+pub enum DrlProximity {
+    Nearest = 0,
+    Nearer = 1,
+    Near = 2,
+    Nearish = 3,
+}
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
pub enum DrlProximity {
    Nearest,
    Nearer,
    Near,
    Nearish,
}
```

We don't actually use `from_repr`.